### DISCUSSION TITLE
[Feature Request]Collapsible segments

### DISCUSSION THREAD

**Discussion by ShadowFM** [fomantic/Fomantic-UI#1621] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Like the subject is a collapsible segment would be great with a plus as icon or something to hide or show something within the segment

**Comment by exoego** [comment, marked as answer]
Does accordion not satisfy your usecase? 
https://fomantic-ui.com/modules/accordion.html

**Comment by ShadowFM** [comment]
I thought and tried it too but then the other "sections" will be closed. My usecase is to open or close another section which will not close the other ones

**Comment by exoego** [comment]
> My usecase is to open or close another section which will not close the other ones

For such usecase, it is required to configure `exclusive` to `false`. 

```javascript
$('.ui.accordion').accordion({
  exclusive: false
});
```

Other settings can be found in https://fomantic-ui.com/modules/accordion.html#/settings

**Comment by lubber-de** [comment]
Here is a working jsfiddle according to your needs using the `exclusive` approach @exoego mentioned above
I adjusted some custom CSS, so it uses a plus/minus icon instead of the default dropdown arrow

https://jsfiddle.net/lubber/jh3a90vo/13/
![segment_accordion_plusminus](https://user-images.githubusercontent.com/18379884/89022617-99266300-d322-11ea-98cd-7443bd20b9cc.gif)

**Comment by lubber-de** [comment]
Closing because solutions with existing fomantic features are available and provided
